lib/osf-components/addon/components/node-categories/component.ts

```diff
@@ -0,0 +1,60 @@
+import { action, computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import Toast from 'ember-toastr/services/toast';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import Node, { NodeCategory } from 'ember-osf-web/models/node';
+import { CategoryManager } from 'osf-components/components/editable-field/category-manager/component';
+import template from './template';
+
+const categoryToIconMap: Record<NodeCategory, string> = {
+    analysis: 'bar-chart',
+    communication: 'comment',
+    data: 'database',
+    hypothesis: 'lightbulb-o',
+    instrumentation: 'flask',
+    'methods and measures': 'pencil',
+    procedure: 'cogs',
+    project: 'cube',
+    software: 'laptop',
+    other: 'th-large',
+    uncategorized: 'circle-o-notch',
+};
+
+@layout(template)
+export default class NodeCategories extends Component {
+    manager!: CategoryManager;
+    node!: Node;
```

Comment:
If you're using a manager, you shouldn't pass the node too. It looks like you did this so you could use the same component for both display and edit, which is nice, but I think it'd be cleaner to split the display into another component. Something like:
- `<NodeCategory @category={{node.category}} />`
  - displays icon and category name
  - owns mapping from category to icon
- `<NodeCategoryPicker @manager={{manager}} />`
  - displays select box, communicates selection to manager
  - uses `<NodeCategory>` to display each menu item

Then the editable field invocation becomes more like
```hbs
<EditableField ... as |field|>
  <field.edit>
    <NodeCategoryPicker @manager={{field.manager}} />
  </field.edit>
  <field.display>
    <NodeCategory @category={{field.manager.category}} />
  </field.display>
</EditableField>
```
and this component doesn't need to switch behavior based on whether a manager is present.